src/ros_video_components/src/ros_coord_bearing.cpp

```diff
@@ -0,0 +1,66 @@
+﻿#include "ros_video_components/ros_coord_bearing.hpp"
+
+ROSCoordBearing::ROSCoordBearing(QQuickItem *parent) : QQuickPaintedItem(parent),
+          nh(), mapSubscriber(nh, "map", 1), tfFilter(mapSubscriber, tfListener, "base_link", 1)
```

Comment:
It would be good for these frames to be set as qml variables that can be configured rather than hardcoded (have a look at how the video component does that)

Reply:
Not sure how to do this, or what you mean by this. Can you give an example, or say how this should be written, in the interest of time?